pkg/apiserver/config/config.go

```diff
@@ -104,6 +105,7 @@ type Config struct {
 	NotificationOptions   *notification.Options                      `json:"notification,omitempty" yaml:"notification,omitempty" mapstructure:"notification"`
 	KubeEdgeOptions       *kubeedge.Options                          `json:"kubeedge,omitempty" yaml:"kubeedge,omitempty" mapstructure:"kubeedge"`
 	MeteringOptions       *metering.Options                          `json:"metering,omitempty" yaml:"metering,omitempty" mapstructure:"metering"`
+	GatewayOptions        *gateway.Options                           `json:"gateway,omitempty" yaml:"router,omitempty" mapstructure:"router"`
```

Comment:
Should we use `gateway` for the YAML as well? I am a little worried about the difference here

Reply:
Yes, it should be `gateway`. fixed.